components/QuickStartMenu.tsx

```diff
@@ -1,15 +1,58 @@
 // Global
+import React, { useState, useRef, useEffect } from 'react';
 import { TTailwindString } from 'tailwindcss-classnames';
 import { classnames } from 'tailwindcss-classnames';
+// Local
+import { SitecoreQuickLinks } from '@/data/data-navigation';
 
 type QuickStartMenuProps = {
   className?: TTailwindString;
+  callback?: () => void;
 };
 
-const QuickStartMenu = ({ className }: QuickStartMenuProps): JSX.Element => {
+const QuickStartMenu = ({ className, callback }: QuickStartMenuProps): JSX.Element => {
+  const quickStartMenuRef = useRef<HTMLDivElement>(null);
+  const [isOpen, setOpen] = useState(false);
+  const toggleQuickStartMenu = (event: React.MouseEvent) => {
+    if (callback && (event?.target as HTMLButtonElement)?.localName !== 'button') {
+      console.log(event.target);
+      callback();
+    }
```

Comment:
Is there any reason for the callback here? It seems like this button is pretty internal to the component. Curious if we have a use case for something using that prop.